### PR TITLE
[WIP] Remove rebass on front page, centering not working

### PR DESCRIPTION
**Pull request by chtinahow**
image
Currently looking like this. I'm having issues getting it to center in the middle of the page correctly

### CODE REVIEW

**Comment by chtinahow** [comment]
New view: 
image

**Comment by billdybas** [comment]
I just glanced at your code - didn't checkout your branch.

Yea, this is on the right track. We want to remove all instances where we use `rebass` to minimize dependencies and keep styling more cohesive.

This means, anywhere we use `rebass` features like `<Flex>` and `<Box>` (from `grid-styled`), convert them into the equivalent Reactstrap (i.e. `<Row>`, `<Col>`) or raw CSS code.

When you were talking about this in person, you made it seem like you were hung up on choosing between Reactstrap vs. raw CSS (i.e. flexbox). Reactstrap's (aka Bootstrap 4's) grid system _**is**_ flexbox – the `<Row>` and `<Col>` components are just flexbox wrappers that provide some convenience and consistency around styling at different breakpoints. 

`<Row>` and `<Col>` should be the starting point when creating a new layout, but feel free to deviate as necessary and drop down to raw CSS if the layout doesn't fit perfectly into the Bootstrap mold. The point of using Bootstrap in this project is to make layout generally consistent across the codebase and to reduce having to write all the flexbox layout code ourselves.

I honestly don't care how you go about switching over from `rebass` provided the pages look and respond the same as before (and you're _not_ using CSS grids, per our previous discussion – still too experimental). Use your best judgement and just ship, ship, ship!

http://getbootstrap.com/docs/4.0/layout/grid/
https://reactstrap.github.io/components/layout/

File: frontend/src/Home/Page.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Row, Col } from 'reactstrap'
+import { Container, Row, Col } from 'reactstrap'
 import { Flex } from 'rebass'
```
Comment (robmcl4) [inline frontend/src/Home/Page.js]: unused import?

File: frontend/src/Home/Page.js [hunk 2]
```diff
@@ -37,16 +43,17 @@ const LoginButton = styled.a`
     text-decoration: none;
   }
 `
-
+// .col .col-sm-12 .col-md-6 .col-md-offset-3
```
Comment (kayladavis) [inline frontend/src/Home/Page.js]: Why are these comments here?

**Comment by billdybas** [comment]
Closes #157

**Pull request closed without merging**